curated_transformers/models/falcon/layer.py

```diff
@@ -118,7 +118,7 @@ def forward(
             corresponding mask element is set to ``False`` are ignored
             during attention calculation.
 
-            *Shape:* ``(batch_size, seq_len)``
+            *Shape:* ``(batch_size, 1, 1, seq_len)``
```

Comment:
Maybe we should consider removing the shape from attention mask arguments? I think the shape makes sense for `Tensor` arguments. But for `AttentionMask` the shape is already checked/enforced by the class itself. Having the shape in the argument doc suggest that this is something that the user should take care of.

Reply:
Fair point. Removed.